cluster-operator/src/main/java/io/strimzi/operator/cluster/model/cruisecontrol/Capacity.java

```diff
@@ -182,4 +216,105 @@ public static Double getThroughputInKiB(String throughput) {
         String size = throughput.substring(0, throughput.indexOf("B"));
         return StorageUtils.convertTo(size, "Ki");
     }
+
+    private void processCapacityEntries(BrokerCapacity bc, Storage s) {
+        Integer cpu = Broker.DEFAULT_CPU_UTILIZATION_CAPACITY;
+        String disk = processDisk(storage, Broker.DEFAULT_BROKER_ID);
+        Double inboundNetwork = processInboundNetwork(bc, null);
+        Double outboundNetwork = processOutboundNetwork(bc, null);
+
+        // Default broker entry
+        Broker defaultBroker = new Broker(Broker.DEFAULT_BROKER_ID, cpu, disk, inboundNetwork, outboundNetwork);
+        capacityEntries.put(Broker.DEFAULT_BROKER_ID, defaultBroker);
+
+        if (storage instanceof JbodStorage) {
+            // A capacity configuration for a cluster with a JBOD configuration
+            // requires a distinct broker capacity entry for every broker because the
+            // Kafka volume paths are not homogeneous across brokers and include
+            // the broker pod index in their names.
+            inboundNetwork = processInboundNetwork(bc, null);
+            outboundNetwork = processOutboundNetwork(bc, null);
+            for (int id = 0; id < replicas; id++) {
+                disk = processDisk(storage, id);
+                Broker broker = new Broker(id, cpu, disk, inboundNetwork, outboundNetwork);
+                capacityEntries.put(id, broker);
+            }
+        }
+
+        if (bc != null) {
+            // For checking for duplicate brokerIds
+            HashSet<Integer> overrideIds = new HashSet<>();
+            List<BrokerCapacityOverride> overrides = bc.getOverrides();
+            // Override broker entries
+            if (overrides != null) {
+                for (BrokerCapacityOverride override : overrides) {
+                    List<Integer> ids = override.getBrokers();
+                    inboundNetwork = processInboundNetwork(bc, override);
+                    outboundNetwork = processOutboundNetwork(bc, override);
+                    for (Integer id : ids) {
+                        disk = processDisk(storage, id);
+                        if (capacityEntries.containsKey(id)) {
+                            if (overrideIds.contains(id)) {
+                                LOGGER.warnOp("Duplicate broker id %d found, using first occurrence." + id);
+                            } else {
+                                overrideIds.add(id);
+                            }
+                            Broker broker = capacityEntries.get(id);
+                            broker.setInboundNetworkKiBPerSecond(inboundNetwork);
+                            broker.setOutboundNetworkKiBPerSecond(outboundNetwork);
```

Comment:
Test to make sure reference works (so that we don't have to explicitly add this broker object to the capacityEntries HashMap)